README.md

```diff
@@ -98,6 +98,14 @@ You can configure golangci-lint with `go.lintFlags`, for example to show issues
   "go.lintFlags": ["--enable-all", "--new"],
 ```
 
+An alternative of golint is [revive](https://github.com/mgechev/revive). It extensible, configurable, provides superset of the rules of golint rules, and has significantly better performance.
+
+To configure revive, use:
+
+```javascript
+  "go.lintFlags": ["-config=[PATH_TO_CONFIG.toml]"]
```

Comment:
Users might want to use the `${workspaceFolder}` or `${workspaceRoot}` variables for the config file. We support this for the config file used by `gometalinter`. You might want to do something similar for revive.

See https://github.com/Microsoft/vscode-go/blob/master/src/goLint.ts#L67

Reply:
Just updated the readme & tested the `${workspaceRoot}`/`${workspaceFolder}` support locally. Everything seems to be functioning correctly.